src/modules/ipfix/aggregator/BaseHashtable.cpp

```diff
@@ -480,6 +480,12 @@ void BaseHashtable::postReconfiguration()
 	sendDataTemplate();
 }
 
+void BaseHashtable::clearStatistics()
+{
+	statExportedBuckets = 0;
+	statLastExpBuckets = 0;
+}
```

Comment:
statRecordsReceived, statRecordsSent dded and pushed, thanks.

statTotalEntries gets already incremented _and_ decremented as the table changes, so as I mentioned I think it shouldn't be resettable. What do you think?
https://github.com/tumi8/vermont/blob/master/src/modules/ipfix/aggregator/BaseHashtable.cpp#L329